asterius/test/rts/MVar.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE LambdaCase #-}
+
+import Asterius.Types
+import Data.Coerce
+import Control.Concurrent
+import Control.Concurrent.MVar
+import Control.Monad
+import System.Mem
+
+foreign import javascript "console.log(${1})" js_print :: JSVal -> IO ()
+
+printString :: String -> IO ()
+printString s = js_print (coerce (toJSString s))
+
+main :: IO ()
```

Comment:
I suggest inserting `performGC` calls to make sure the `MVar` objects are properly handled by the gc.

Reply:
Currently the GC doesn't handle resurrected threads, hence we don't get any "indefinitely blocked on MVar exceptions". I'll add a ticket for this.

Reply:
Done #310